squad/frontend/views.py

```diff
@@ -58,9 +58,13 @@ def build(request, group_slug, project_slug, version):
     group = Group.objects.get(slug=group_slug)
     project = group.projects.get(slug=project_slug)
     build = project.builds.prefetch_related('test_runs', 'test_runs__status', 'test_runs__status__suite', 'test_runs__status__test_run__environment').get(version=version)
+    metadata_dict = {}
+    for testrun in build.test_runs.all():
+        metadata_dict.update(json.loads(testrun.metadata_file))
```

Comment:
do we want to include *all* metadata entries in the build metadata? I assume several of the items will be specific to each test run?

maybe the build metadata should be the _intersection_ of the test run metadata?